data/es.properties

```diff
@@ -1,3 +1,3 @@
-app_title = Aplicación vacía privilegiada
+app-title = Aplicación vacía privilegiada
```

Comment:
there’s no rule for that. You can use snake_case, camelCase, or hyphens if you really have no taste. :-D